From a7984302697cf829c512d510562df8c80bf8521c Mon Sep 17 00:00:00 2001
From: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Date: Fri, 18 Oct 2024 09:39:55 +0200
Subject: [PATCH 762/954] feat(st-clock):rename RCC_USBTCCFGR register into
 RCC_UCPDCFGR

Rename this register to be aligned with the reference manual.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git dc7a087914af285572c65c82c2c8fa993b643002 feat(st-clock):rename RCC_USBTCCFGR register into RCC_UCPDCFGR]
Signed-off-by: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Change-Id: Ia10c287bf4068742a7add9016c1a87e300eebff0
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/412465
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
Reviewed-by: Maxime MERE <maxime.mere@foss.st.com>
---
 drivers/st/clk/clk-stm32mp2.c      |  2 +-
 include/drivers/st/stm32mp25_rcc.h | 10 +++++-----
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/drivers/st/clk/clk-stm32mp2.c b/drivers/st/clk/clk-stm32mp2.c
index ec6c05104e..86f8d8a306 100644
--- a/drivers/st/clk/clk-stm32mp2.c
+++ b/drivers/st/clk/clk-stm32mp2.c
@@ -607,7 +607,7 @@ static const struct gate_cfg gates_mp2[LAST_GATE] = {
 	GATE_CFG(GATE_STGEN,		RCC_STGENCFGR,		1,	0),
 #if !STM32MP21
 	GATE_CFG(GATE_USB3PCIEPHY,	RCC_USB3PCIEPHYCFGR,	1,	0),
-	GATE_CFG(GATE_USBTC,		RCC_USBTCCFGR,		1,	0),
+	GATE_CFG(GATE_USBTC,		RCC_UCPDCFGR,		1,	0),
 	GATE_CFG(GATE_I2C8,		RCC_I2C8CFGR,		1,	0),
 #endif /* !STM32MP21 */
 	GATE_CFG(GATE_OSPI1,		RCC_OSPI1CFGR,		1,	0),
diff --git a/include/drivers/st/stm32mp25_rcc.h b/include/drivers/st/stm32mp25_rcc.h
index 13cf5412db..5b6c527196 100644
--- a/include/drivers/st/stm32mp25_rcc.h
+++ b/include/drivers/st/stm32mp25_rcc.h
@@ -429,7 +429,7 @@
 #define RCC_USB3DRCFGR				U(0x808)
 #define RCC_USB3PCIEPHYCFGR			U(0x80C)
 #define RCC_PCIECFGR				U(0x810)
-#define RCC_USBTCCFGR				U(0x814)
+#define RCC_UCPDCFGR				U(0x814)
 #define RCC_ETHSWCFGR				U(0x818)
 #define RCC_ETHSWACMCFGR			U(0x81C)
 #define RCC_ETHSWACMMSGCFGR			U(0x820)
@@ -3484,10 +3484,10 @@
 #define RCC_PCIECFGR_PCIELPEN			BIT(2)
 #define RCC_PCIECFGR_PCIESTPEN			BIT(4)
 
-/* RCC_USBTCCFGR register fields */
-#define RCC_USBTCCFGR_USBTCRST			BIT(0)
-#define RCC_USBTCCFGR_USBTCEN			BIT(1)
-#define RCC_USBTCCFGR_USBTCLPEN			BIT(2)
+/* RCC_UCPDCFGR register fields */
+#define RCC_UCPDCFGR_UCPDRST			BIT(0)
+#define RCC_UCPDCFGR_UCPDEN			BIT(1)
+#define RCC_UCPDCFGR_UCPDLPEN			BIT(2)
 
 /* RCC_ETHSWCFGR register fields */
 #define RCC_ETHSWCFGR_ETHSWRST			BIT(0)
-- 
2.53.0

